.github/PULL_REQUEST_TEMPLATE/pr_external.md

```diff
@@ -10,6 +10,8 @@
 
 - [ ] Added e2e tests
 
+- [ ] Added json mocks for any new views?
```

Comment:
Within our `test` directory, we have `e2e`, `testcafe`, and `unit` types. Here, let's couple this to `testcafe` to be extremely verbose:

 ```suggestion
- [ ] Added testcafe tests with JSON mocks for any new views
```

Reply:
Also - can we cleanup the double newline spaces between these items and remove the `?` from the **unit** label?

Reply:
Fixed